### PR TITLE
Added way to get Atoms in the highest/lowest bin of the ImportanceIndex

### PR DESCRIPTION
**Pull request by rTreutlein**
This is needed for cheaply figuring out min and max STI values.

### CODE REVIEW

**Comment by rTreutlein** [comment]
Didn't actually want both these commits in this pull request. But if there is no issue with either one we can just merge like this. If there is ill just have to rewrite history on my branch.

File: opencog/atomspace/ImportanceIndex.cc [hunk 1]
```diff
@@ -123,3 +146,35 @@ UnorderedHandleSet ImportanceIndex::getHandleSet(
...
+        }
+    }
+
+    return ret;
+}
+
+UnorderedHandleSet ImportanceIndex::getMinBinContents()
+{
+    UnorderedHandleSet ret;
+    for (AtomSet s : idx)
```
Comment (linas) [inline opencog/atomspace/ImportanceIndex.cc]: I am guessing that what you really want is to get rid of the for-loop, and just set
```
const AtomSet& s = idx[0];
```
and for the other one, 
```
const AtomSet& s = idx[IMPRTANCE_MAX_BINS-1];
```

Comment (rTreutlein) [inline opencog/atomspace/ImportanceIndex.cc]: What I want to get is the smallest/biggest bin that is not empty. So iterate over all the bins starting from the bottom/top and if I find a bin that is not empty I immediately return that. Maybe you missed the return at line 159.

**Comment by rTreutlein** [comment]
@linas: Yes this is an exponential bin size setup. But so that the size doesn't grow too quickly each binSize is part of a group that contains (in this case) 8 bins with this Size.

This bias the Index towards lower STI values and allows us to initialize the STI with a small random value so that not all new Atoms end up in the same bin.

Why exactly do you think this is not going to work? It seemed to do what I wanted it to when I was experimenting with it.

File: opencog/atomspace/ImportanceIndex.cc (hunk 1 above)
Comment (linas) [inline opencog/atomspace/ImportanceIndex.cc]: yes I did. OK

**Pull request merged**